### DISCUSSION TITLE
Accessing postgres database directly?

### DISCUSSION THREAD

**Discussion by sanketch** [supabase/supabase#413] [Questions] [answered]
Is there a way to directly access postgres database without going through the Supabase API? 

Essentially, I would like the ability to write contents to the database through the Supabase API, but for reads, I would like to use Cube.js which requires connection to the postgres database directly. Is there a way to receive the database creds/access?

**Comment by steve-chavez** [comment, marked as answer]
Yes, the postgres connection string is at `/project/<your_id>/settings/database`(click cog, then database).

**Reply by sanketch** [reply]
Ahh wow I missed that. Thank you for the quick response!
